app/frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -32,6 +32,8 @@ export default function Avatar({ user, grow, ...otherProps }: AvatarProps) {
       {...otherProps}
       className={classNames(classes.root, { [classes.grow]: grow })}
       src={user.avatarUrl}
-    />
+    >
+      {user.name.split(" ").map((name) => name.charAt(0))}
```

Comment:
I would split on sequences of whitespace and use index notation.

```suggestion
      {user.name.split(/\s+/).map((name) => name[0])}
```